experimenter/tests/integration/nimbus/ios/test_setup_ios_integration.py

```diff
@@ -0,0 +1,74 @@
+import json
+import logging
+import time
+from pathlib import Path
+
+import pytest
+import requests
+from nimbus.models.base_dataclass import BaseExperimentApplications
+from nimbus.pages.experimenter.summary import SummaryPage
+from nimbus.utils import helpers
+
+
+@pytest.fixture
+def experiment_slug():
+    return "firefox-ios-integration-test"
+
+
+@pytest.fixture
+def default_data_api(application):
+    feature_config_id = helpers.get_feature_id_as_string(
+        "messaging", BaseExperimentApplications.FIREFOX_IOS.value
+    )
+    return {
+        "hypothesis": "Test Hypothesis",
+        "application": application,
+        "changelogMessage": "test updates",
+        "targetingConfigSlug": "no_targeting",
+        "publicDescription": "Some sort of Fancy Words",
+        "riskRevenue": False,
+        "riskPartnerRelated": False,
+        "riskBrand": False,
+        "riskMessage": False,
+        "featureConfigIds": [int(feature_config_id)],
+        "referenceBranch": {
+            "description": "reference branch",
+            "name": "control",
+            "ratio": 50,
+            "featureValues": [
+                {
+                    "featureConfig": str(feature_config_id),
+                    "value": "{}",
+                },
+            ],
+        },
+        "treatmentBranches": [],
+        "populationPercent": "100",
+        "totalEnrolledClients": 55,
+        "firefoxMinVersion": "FIREFOX_120",
+    }
```

Comment:
If we're running in the Nimbus test context can't we just reuse the existing fixture for this? 

https://github.com/mozilla/experimenter/blob/main/experimenter/tests/integration/nimbus/conftest.py#L368-L397

Reply:
You can overwrite fields if you need to like

```py
default_data_api['featureConfigs'] = [whatever.id]
```